### ISSUE TITLE
[Docs?]: add missing visual and code examples on Icon page

### ISSUE THREAD

**Issue by caseybaggz** [pluralsight/tva#373]
### Latest version

- [X] I have checked the latest version

### Summary 💡

The HS/Icon.mdx has some missing visual and code examples needed in the a11y section. There are "TODO" notes marked where things should be added on the page.
